VMWare/vmware-esxi/tests/AUTH_SSH_PASSWORD.json

```diff
@@ -0,0 +1,34 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "info hostd[2099656] [Originator@6876 sub=Default opID=esxcli-76-ecbf] Accepted password for user root from 127.0.0.1"
+  },
+  "expected": {
+    "message": "info hostd[2099656] [Originator@6876 sub=Default opID=esxcli-76-ecbf] Accepted password for user root from 127.0.0.1",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": ["process", "session", "authentication"],
+      "type": [
+        "info"
+      ]
```

Comment:
I think that `event.category` = `configuration` and `event.type` = `change` will be more suitable.